ola/Makefile.mk

```diff
@@ -18,6 +18,7 @@ lib_LTLIBRARIES += ola/libola.la
 ola_libola_la_SOURCES = \
     ola/AutoStart.cpp \
     ola/ClientRDMAPIShim.cpp \
+    ola/ClientTypesFactory.cpp \
```

Comment:
You need to add the .h file here as well. Otherwise it won't be included in the tarball, which is what travis tries to build.